libraries/client/client_webhook.py

```diff
@@ -388,6 +400,31 @@ def add_payload_to_tx_converter(self, callback_url, identifier, payload, rc, sou
                 job = json_data['job']
         return identifier, job
 
+    def send_lint_request_to_tx_manager(self, job, repo_url):
+        payload = {
+            'job_id': job['job_id'],
+            'resource_type': job['resource_type'],
+            'file_type': job['input_format'],
```

Comment:
I will also need the language code.  If not english I have to check chapter headings to make sure they were translated.